keps/sig-storage/5040-remove-gitrepo-driver/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: Disable hooks for gitRepo volumes by default.
+kep-number: 5040
+authors:
+- "@vinayakankugoyal"
+owning-sig: sig-storage
+participating-sigs:
+status: implementable
+creation-date: 2024-12-14
+reviewers:
+- "@thockin"
+- "@liggitt"
+- "@saad-ali"
+approvers:
+- "@thockin"
+
+see-also:
+
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
```

Comment:
Check that this is right. You might mean “deprecated”.

Reply:
I wasn't sure what values this can take.

But based on your feedback about using “deprecated” in the graduation criteria maybe it makes sense to change this to "removed with opt-out"?

Reply:
We had a discussion on adding Deprecated but that PR didn't merge, let me go look for it.

Reply:
Turns out it was waiting for me.  Oops.  https://github.com/kubernetes/enhancements/pull/4898

Since we are jumping to off-by-default, I think the Stage you want is "disabled"

Reply:
Gotcha, Ill wait for https://github.com/kubernetes/enhancements/pull/4898 to be merged before updating this.

Reply:
AFAICT nothing is validating this, so don't block on it :)

Reply:
Done.

Reply:
😭 

```
"map[/home/prow/go/src/github.com/kubernetes/enhancements/keps/sig-storage/5040-remove-gitrepo-driver/kep.yaml:[getting PRR approver for diabled stage: invalid stage: diabled, should be one of [alpha beta stable]]]"
```

Reply:
Opened https://github.com/kubernetes/enhancements/pull/5049 to fix validation.